packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardTag.tsx

```diff
@@ -1,18 +1,16 @@
-import React, { FC } from 'react';
+import React from 'react';
 import { Text } from '@deriv/quill-design';
 
 type TJurisdictionCardTagProps = {
     tag: string;
 };
 
-const JurisdictionCardTag: FC<TJurisdictionCardTagProps> = ({ tag }) => {
-    return (
-        <div className='absolute top-50 left-50 w-full p-400 text-center rounded-t-[13px] rounded-b-50 bg-[#dfeaff]'>
-            <Text bold className='text-random-blue' size='sm'>
-                {tag}
-            </Text>
-        </div>
-    );
-};
+const JurisdictionCardTag = ({ tag }: TJurisdictionCardTagProps) => (
+    <div className='absolute top-50 left-50 w-full p-400 text-center rounded-t-[13px] rounded-b-50 bg-[#dfeaff]'>
```

Comment:
@arshad-rao-deriv please add this bg color to the tailwind.config file

Reply:
Updated in the Jurisdiction Modal [PR](https://github.com/binary-com/deriv-app/pull/12888).